examples/browser-webrtc/server/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+authors = ["Doug Anderson <douganderson444@peerpiper.io>"]
+edition = "2021"
+license = "MIT"
+name = "webrtc-example-server"
+publish = false
+rust-version = { workspace = true }
+version = "0.1.0"
+
+[[bin]]
+name = "webrtc-example-server-bin"
+path = "src/bin/webrtc-example-server.rs"
+
+[dependencies]
+anyhow = "1.0.72"
+axum = "0.6.19"
+env_logger = "0.10"
+futures = "0.3.28"
+libp2p-core = { workspace = true, features = [] }
+libp2p-identify = { workspace = true }
+libp2p-identity = { workspace = true, features = ["ed25519", "peerid"] }
+libp2p-ping = { workspace = true }
+libp2p-relay = { workspace = true }
+libp2p-swarm = { workspace = true, features = ["macros", "tokio"] }
+libp2p-webrtc = { workspace = true, features = ["tokio"] }
```

Comment:
All our other examples depend on `libp2p` directly and activate the necessary features. Can we do the same thing here?

Reply:
Good point, fixed it.